openshift/openshift-ansible-contrib
# Add end to end openstack testing to travis

#### What does this PR do?

This is using an RDO Cloud account dedicated to CI.

#### How should this be manually tested?
No manual testing. Just check the Travis CI output. There should be the python 2.7 and 3.5 tox jobs, but in addition, a new end to end job that runs the openstack provisioning, too.

The run takes about 40 minutes on average and should only be executed when we change the code in roles or the playbooks/provisioning section.

To see an example of a successful run, go here:

https://travis-ci.org/tomassedovic/openshift-ansible-contrib/jobs/264409141

#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
cc:  @bogdando @cooktheryan @celebdor PTAL
